<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Coal_Sort_Manage</class>
 <widget class="QDialog" name="Coal_Sort_Manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>煤种添加</string>
  </property>
  <widget class="QLabel" name="ticket_price_label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择计价方式</string>
   </property>
  </widget>
  <widget class="QLabel" name="coal_name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入煤种名称</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sell_compute_way">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="purchase_compute_way">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ticket_price_label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择计价方式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_date_btn">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>20</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择添加日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="ticket_price_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入煤种进价</string>
   </property>
  </widget>
  <widget class="QLabel" name="ticket_price_label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入煤种售价</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_value_content">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coal_sort_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="purchase_price">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sell_price">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="modify_detail_list">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>256</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="modify_label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>最近添加项</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>purchase_compute_way</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Coal_Sort_Manage</receiver>
   <slot>on_purchase_compute_way_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sell_compute_way</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Coal_Sort_Manage</receiver>
   <slot>on_sell_compute_way_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_date_btn</sender>
   <signal>clicked()</signal>
   <receiver>Coal_Sort_Manage</receiver>
   <slot>on_add_new_coal_date_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirm</sender>
   <signal>clicked()</signal>
   <receiver>Coal_Sort_Manage</receiver>
   <slot>add_new_coal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>Coal_Sort_Manage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_purchase_compute_way_selected()</slot>
  <slot>on_sell_compute_way_selected()</slot>
  <slot>on_add_new_coal_date_selected()</slot>
  <slot>add_new_coal()</slot>
 </slots>
</ui>
